static/app/views/monitors/components/processingErrors/processingErrorItem.tsx

```diff
@@ -0,0 +1,85 @@
+
+interface Props {
+  checkin: React.ReactNode;
+  error: ProcessingError;
+}
+
+export function ProcessingErrorItem({error, checkin}: Props) {
+  switch (error.type) {
+    case ProcessingErrorType.CHECKIN_ENVIRONMENT_MISMATCH:
+      return tct(
+        'A [checkin] update was sent that did not match the original environment of [env].',
+        {checkin, env: error.existingEnvironment}
+      );
+    case ProcessingErrorType.CHECKIN_FINISHED:
+      return tct(
+        'A [checkin] update was sent to a checkin that already reached a terminal state.',
+        {checkin}
+      );
+    case ProcessingErrorType.CHECKIN_GUID_PROJECT_MISMATCH:
+      return tct(
+        'The guid supplied with a [checkin] matched a checkin for a different project than the one provided in the DSN.',
+        {checkin}
+      );
+    case ProcessingErrorType.CHECKIN_INVALID_DURATION:
+      return tct('A [checkin] was sent with an invalid duration: [duration].', {
+        checkin,
+        duration: error.duration,
+      });
+    case ProcessingErrorType.CHECKIN_INVALID_GUID:
+      return tct('A [checkin] was sent with an invalid GUID', {checkin});
+    case ProcessingErrorType.CHECKIN_VALIDATION_FAILED:
+      return tct('A [checkin] was sent with an invalid format.', {checkin});
+    case ProcessingErrorType.MONITOR_DISABLED:
+      return tct(
+        'A [checkin] was sent, and discarded because this monitor was disabled.',
+        {checkin}
+      );
+    case ProcessingErrorType.MONITOR_DISABLED_NO_QUOTA:
+      return tct(
+        'A [checkin] was sent, but dropped due to this monitor being disabled because of insufficient quota.',
+        {checkin}
+      );
+    case ProcessingErrorType.MONITOR_INVALID_CONFIG:
+      return tct(
+        'A monitor failed to upsert from a sent [checkin] due to an invalid configuration.',
+        {checkin}
+      );
+    case ProcessingErrorType.MONITOR_INVALID_ENVIRONMENT:
+      return tct('A [checkin] was sent with an invalid environment due to: [reason].', {
+        checkin,
+        reason: error.reason,
+      });
+    case ProcessingErrorType.MONITOR_LIMIT_EXCEEDED:
+      return t(
+        'The maximum number of monitors for this project has been exceeded, and no more may be upserted.'
+      );
+    case ProcessingErrorType.MONITOR_NOT_FOUND:
+      return tct(
+        'A [checkin] was sent for a monitor which does not exist. If you meant to upsert a new monitor please provide a configuration with your [checkin]',
+        {checkin}
+      );
+    case ProcessingErrorType.MONITOR_OVER_QUOTA:
+      return tct(
+        'A [checkin] was sent, but dropped due to this monitor being disabled.',
+        {
+          checkin,
+        }
+      );
+    case ProcessingErrorType.MONITOR_ENVIRONMENT_LIMIT_EXCEEDED:
+      return tct(
+        'A sent [checkin] was dropped due to being for a new environment over the limit of allowed environments.',
```

Comment:
```suggestion
        'A [check-in] was sent but dropped because the monitor has already reached the limit of allowed environments. Remove an existing environment to create new ones.',
```